src/main/webapp/index-script.js

```diff
@@ -162,24 +163,31 @@ function handleSearchResults(results, service) {
   // Center on the queried location
   if (results.length > 0) { 
     map.setCenter(results[0].geometry.location);
+    console.log(results.length);
```

Comment:
Is this for temporary debugging? If intended to be submitted, maybe log what this is.